code.sh

```diff
@@ -60,6 +60,13 @@ for i in "${SDK[@]}"; do
   fi
 done
 
+# Create /etc/static/shells before than /etc/shells, because /etc/shells ls linkd etc/static/shells
+# when use nixos is default has't /etc/static/shells
+if [ ! -f /etc/static/shells ]; then
+  mkdir -p /etc/static/
+  printf '/usr/bin/%s\n' sh bash > /etc/static/shells
```

Comment:
It would be better to create a link to the actual file on the NixOS host, rather than creating a new one in the sandbox

```sh
ln -s /var/run/host/etc/static/shells /etc/static/shells
```

Reply:
it's better not to, because it needs to be executed manually by the user, which is inconvenient.

Reply:
What do you mean created manually? Just replace the `printf` command with the `ln` command I provided, and it will be executed on startup the same way. The folder already exists on the host, and is available in the sandbox via that path ([host's `/etc` directory is mounted to `/var/run/host/etc` on the sandbox](https://docs.flatpak.org/en/latest/sandbox-permissions.html#filesystem-access)). It's better as it'll properly represent the available shells